packages/expo-dev-menu/ios/DevMenuWindow.swift

```diff
@@ -2,25 +2,111 @@
 
 import UIKit
 
-class DevMenuWindow: UIWindow {
-  private let manager: DevMenuManager
+extension UIView {
+  func firstSubView<T: UIView>(ofType type: T.Type) -> T? {
```

Comment:
```suggestion
  func firstSubview<T: UIView>(ofType type: T.Type) -> T? {
```
Also I'd feel more comfortable with not using the extension when there is another solution like a top-level private function 😅